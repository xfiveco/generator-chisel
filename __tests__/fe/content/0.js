const fs = require('fs-extra');

const { defaultAnswers, md, postPageSimple } = require('./helpers');

describe('FE Content', () => {
  test('Simple', async () => {
    await global.chiselTestHelpers.generateProjectWithAnswers(
      ['create', '--skip-format-and-build'],
      defaultAnswers,
    );

    fs.outputFileSync('./content/hello.md', md('Hello', 'Some Content'));
    fs.outputFileSync('./src/templates/post.twig', postPageSimple());

    await global.chiselTestHelpers.runChiselScript(['build']);

    await global.chiselTestHelpers.expectFilesToMatchSnapshot(['./dist']);
    await global.chiselTestHelpers.fileMatchesSnapshot('./dist/hello.html');
  });
});
