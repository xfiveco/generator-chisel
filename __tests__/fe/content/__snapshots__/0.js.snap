// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FE Content Simple 1`] = `
Array [
  "dist/hello.html",
  "dist/scripts/app.--HASH--.full.js",
  "dist/scripts/app.--HASH--.js",
  "dist/scripts/app.--HASH--.js.map",
  "dist/styles/main.--HASH--.css",
  "dist/styles/main.--HASH--.css.map",
  "dist/styles/main.--HASH--.full.css",
]
`;

exports[`FE Content Simple 2`] = `
"<!DOCTYPE html>
<h1>Hello Hello</h1>
<div>
  <p>Some Content</p>
</div>"
`;
